compiler/x/runtime/VirtualGuardRuntime.cpp

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2000, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ *******************************************************************************/
+
+#include <stdint.h>
+#include "infra/Assert.hpp"
+
+#define IS_32BIT_SIGNED(x)   ((x) == ( int32_t)(x))
+
+extern "C" void _patchingFence16(void *startAddr);
```

Comment:
This can be found in either `compiler/x/i386/runtime/IA32AsmUtil.inc` or `compiler/x/amd64/runtime/AMD64AsmUtil.inc`.